compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -276,13 +285,29 @@ internal actual class ComposeWindow : UIViewController {
             interfaceOrientationState.value = it
         }
 
-        val composeLayer = attachedComposeContext?.composeLayer ?: return
+        attachedComposeContext?.let {
+            updateLayout(it)
+        }
+    }
 
-        val (width, height) = getViewFrameSize()
-        val scale = density.density
+    private fun updateLayout(context: AttachedComposeContext) {
+        context.scene.also {
+            it.density = density
```

Comment:
I think, what `context.scene.also` is redundant.
It may be used as a simple `context.scene`
```Kotlin
context.scene.density = density
context.scene.constraints = ...
```

Reply:
Either are fine for me me, updated